submissions/TUstiugov/html-css-popup/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="./images/popup-icons/favicon.png" type="image/x-icon">
+  <link rel="stylesheet" href="./style.css">
+  <title>HTML-CSS-Popup</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="header__navigation navigation">
+      <ul class="navigation__list flex-container">
+        <li class="navigation__item">
+          <a href="#" class="navigation__link navigation__link_mail">Mail</a>
```

Comment:
Avoid layout shifting of sibling elements on hover or focus. [Watch video](https://share.vidyard.com/watch/XXnGWTWr3icZBH6t2UWjPK?)

Reply:
Done.